skipchain/skipchain.go

```diff
@@ -585,7 +585,7 @@ func (s *Service) AddFollow(add *AddFollow) (*EmptyReply, error) {
 		}
 		log.Lvlf2("%s FollowSearch %s %x", s.ServerIdentity(), add.Conode, add.SkipchainID)
 	case FollowLookup:
-		si := network.NewServerIdentity(cothority.Suite.Point().Null(), network.NewAddress(network.PlainTCP, add.Conode))
+		si := network.NewServerIdentity(cothority.Suite.Point().Null(), network.NewAddress(network.TLS, add.Conode))
```

Comment:
This can't work in heterogeneous networks. I think add.Conode needs to be a ServerIdentifier.